code/src/components/GameState.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+import { Action } from "./Action"
```

Comment:
It should be written like this:

import React from 'react'
import { useSelector } from 'react-redux'

import { Action } from "./Action"